library/network/src/lib/y2firewall/firewalld/service.rb

```diff
@@ -56,6 +56,9 @@ def initialize(name)
       # @return description [String] service long description
       attr_reader :description
 
+      # @return modified [Array<Symbol>] the list of relations modified
+      attr_reader :modified
+
       has_many :ports, :protocols, scope: "service"
```

Comment:
just a note about the model for me
this should cover that a service represented by an object of this class (http server)
can run on multiple ports (say 80, 8080)
and support multiple protocols (say http, https)